package.json

```diff
@@ -51,10 +51,11 @@
   },
   "dependencies": {
     "@cospired/i18n-iso-languages": "2.2.0",
+    "@edx/browserslist-config": "^1.1.0",
```

Comment:
I think we'd want this to be a `devDependency` only. If we're including this, do we also need to set the `"browserslist"` property in the package.json to extend `@edx/browserslist-config`? E.g. (see prior art in other repos):

```json
{
  "browserslist": [
    "extends @edx/browserslist-config"
  ]
}
```